src/lib/modifiers.ts

```diff
@@ -0,0 +1,17 @@
+import { Familiar } from "kolmafia";
+import { $familiar } from "libram";
+
+import { selectBestFamiliar } from "./familiar";
+
+export function getModString(): string {
+  const fam: Familiar = selectBestFamiliar();
+  const banderBoots: boolean =
```

Comment:
I'm cool with this. For bonus points, can break this out as a switch statement on familiar.

e.g.
```
const fwWeight = getModWeight(fam);
. . .

function getModWeight(fam) {
let w = "0.00"
switch(fam) {
case $familiar`bander`: 
w = "0.25"
. . .
return w
}
```
}
}

Reply:
Done.